sdk/eventhub/Azure.Messaging.EventHubs/perf/Azure.Messaging.EventHubs.Perf/Azure.Messaging.EventHubs.Perf/Scenarios/SendEvents.cs

```diff
@@ -0,0 +1,87 @@
+//Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Azure.Messaging.EventHubs;
+using Azure.Messaging.EventHubs.Producer;
+using Azure.Test.Perf;
+
+namespace Azure.Messafging.EventHub.Perf.Scenarios
+{
+    /// <summary>
+    /// The performance test scenario focused on sending events to Azure EventHubs.
+    /// </summary>
+    /// <seealso cref="Azure.Test.Perf.PerfTest{SizeOptions}" />
```

Comment:
This `seealso` is referencing a type that isn't used here.    We should update to reference `EventHubsPerfOptions`